requirements.txt

```diff
@@ -1,3 +1,4 @@
-ops >= 2.2.0
+ops==2.14.0
```

Comment:
question (non-blocking): What is the reason for rolling this back? How long will it stay this way?

Reply:
This is actually updating the version and pinning it to `2.14.0` from `2.2.0`.